cartoframes/contrib/vector.py

```diff
@@ -326,12 +330,17 @@ def vmap(layers, context, size=(1024, 632), basemap=BaseMaps.voyager):
         context (:py:class:`CartoContext <cartoframes.context.CartoContext>`):
           A :py:class:`CartoContext <cartoframes.context.CartoContext>`
           instance
+        size (integer tuple): size in pixels of the generated iframe with the
```

Comment:
Great catch! 

How about `size (tuple of ints)` instead?

And for the description, maybe something that leads them to understand how to structure the size: `A (width, height) pair for the size of the map. Default is '(1024, 632)'.`